spec/active_hash/base_spec.rb

```diff
@@ -404,6 +404,23 @@ class Region < ActiveHash::Base
     end
   end
 
+  describe ".pluck" do
+    before do
+      Country.data = [
+        {:id => 1, :name => "US"},
+        {:id => 2, :name => "Canada"}
+      ]
+    end
+
+    it "returns an two dimensional Array of attributes values" do
+      expect(Country.pluck(:id, :name)).to match_array([[1,"US"], [2, "Canada"]])
+    end
+
+    it "returns an Array of attribute values" do
+      expect(Country.pluck(:id)).to match_array([1,2])
+    end
```

Comment:
@syguer 
Add test for single symbol case.

Reply:
great 👍